### ISSUE TITLE
[ldd-disp] Some regression tests are failing

### ISSUE THREAD

**Issue by wkiri** [pds-data-dictionaries/PDS4-LDD-Issue-Repo#202]
**Describe the issue identified (if applicable)**
The following test labels include an explicit version (1.17.0.0) of the information model, which causes the automated regression tests to fail because they are tested against every version in `pds4_versions.txt`.

* https://github.com/pds-data-dictionaries/ldd-disp/blob/dmsp-test/test/test1_FAIL.xml
* https://github.com/pds-data-dictionaries/ldd-disp/blob/dmsp-test/test/test1_VALID.xml

E.g. the validate logs (for IM version 1.16.0.0) show

>   FAIL: file:/home/runner/work/ldd-disp/ldd-disp/test/test1_VALID.xml
      ERROR  [error.label.schematron]   line 19, 32: The attribute pds:information_model_version must be equal to the value '1.16.0.0'.

**Describe the solution you'd like**
Test labels should be updated to specify
> <information_model_version>[IM_VERSION]</information_model_version>

That way the regression test process will insert the appropriate IM version for each test run.

**Describe alternatives you've considered**
None.

**LDD Dictionary Version**
1.5.1.0

**PDS4 IM Version**
All specified in `pds4_versions.txt`

**Need-by Date**
Whenever you would like to achieve a pass on all regression tests as indicated by a green checkmark (pass) on commits to this repository.

**Comment by acraugh** [comment]
@wkiri, this would be cleaned up as part of the DMSP follow-up, which will replace the current tests and also be adding new regressions tests. @jordanpadams, should we trouble-shoot this now, roll the new regression tests into this ticket, or close this ticket and include the details of this issue in with the other regression test changes? Timeline for completing all updates to the Display namespace is 3-4 weeks. Thanks!

**Comment by wkiri** [comment]
> @wkiri, this would be cleaned up as part of the DMSP follow-up, which will replace the current tests and also be adding new regressions tests.

@acraugh  Yep, and this may apply to other namespaces as well - I haven't looked at them all.  In this case I noticed the failed tests and wanted to capture them here.

**Comment by thareUSGS** [comment]
should be closable once DISPL 1.6.0.0 is merged into main. Still cleaning up tests in that brench. https://github.com/pds-data-dictionaries/ldd-disp/tree/v1600/test

**Comment by thareUSGS** [comment]
lots of good test from DMSP group copied to v1600 with a README.md for tests also.
https://github.com/pds-data-dictionaries/ldd-disp/tree/v1600/test
